pkg/atproto/sync.go

```diff
@@ -423,6 +424,55 @@ func (atsync *ATProtoSynchronizer) handleCreateUpdate(ctx context.Context, userD
 			log.Error(ctx, "failed to create metadata configuration", "err", err)
 		}
 
+	case *streamplace.ModerationPermission:
+		repo, err := atsync.SyncBlueskyRepoCached(ctx, userDID, atsync.Model)
+		if err != nil {
+			return fmt.Errorf("failed to sync bluesky repo: %w", err)
+		}
+		if repo == nil {
+			// someone we don't know about
+			return nil
+		}
```

Comment:
Should be an impossible case if `SyncBlueskyRepoCached` just returned successfully
```suggestion
```